raspberry/bzzz/estimators/altitude_Kalman_filter.py

```diff
@@ -71,64 +85,90 @@ def __update_At(self):
         """Update state matrix At. 
         NOTE: This function should be called whenever Tt is updated.
         """
+        k_t = 0.5*self.__Ts**2
+
         self.__At = np.array([
-            [1, self.__Ts, 0, 0],
+            [1, self.__Ts, k_t*self.__Tt, k_t],
             [0, 1, self.__Ts*self.__Tt, self.__Ts],
             [0, 0, 1, 0],
             [0, 0, 0, 1]
         ])
 
-    def __update_Tt(self, Tt):
+    def __update_Tt(self, Tt, pitch_rad, roll_rad):
         """Update the throttle reference/ system input.
 
         :param Tt: Current normalized throttle reference.
+        :param pitch_rad: current drone pitch in radians.
+        :param roll_rad: current drone roll in radians.
         """
-        self.__Tt = Tt
+        self.__Tt = math.cos(pitch_rad) * math.cos(roll_rad) * float(Tt)
 
-    def measurement_update_cache(self):
+    def MU_cache(self):
```

Comment:
I thought you had renamed this to `measurement_update_cache`. Have you pushed the changes?